h5p-bildetema/src/components/Language/Language.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styles from "./Language.module.scss";
+import { Language as LanguageType } from "../../../../common/types/types";
+
+type SelectProps = {
+  language: LanguageType;
+};
+
+export const Language: React.FC<SelectProps> = ({
+  language: { label, code },
+}) => {
+  return (
+    <div className={styles.language}>
+      <div>
+        <label htmlFor={code} className={styles.input_container}>
+          <input type="checkbox" id={code} />
```

Comment:
A `value` property is probably needed on the Language component and the checkbox.